README.md

```diff
@@ -187,6 +187,43 @@ This should generally always be sent. If you need to disable it, set the `ABS_XS
 define( 'ABS_XSS_PROTECTION_HEADER', false );
 ```
 
+### Restrict CORS origins
+
+By default, WordPress will allow REST API requests from any Origin. This plugin adds a new filter `altis.security.browser.rest_allow_origin` to restrict [CORS origins](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS). 
+
+To completely disallow external requests, set the filter to `false`:
+
+```php
+add_filter( 'altis.security.browser.rest_allow_origin', '__return_false' );
+```
+
+To allow specific origins only:
+```php
+add_filter( 'altis.security.browser.rest_allow_origin', function ( $allow, $origin ) {
```

Comment:
As per my comment on the docs PR, it will be better to specify the types of the parameters and the return type of the function.